source/documentation/administration_guide/topics/Syntax_for_the_engine-backup_Command.adoc

```diff
@@ -13,29 +13,32 @@ The `engine-backup` command works in one of two basic modes:
 ----
 # engine-backup --mode=restore
 ----
-These two modes are further extended by a set of parameters that allow you to specify the scope of the backup and different credentials for the engine database. Run `engine-backup --help` for a full list of parameters and their function.
+These two modes are further extended by a set of options that allow you to specify the scope of the backup and different credentials for the engine database. Run `engine-backup --help` for a full list of options and their function.
 
 *Basic Options*
 
-*`--mode`*:: Specifies whether the command will perform a backup operation or a restore operation. Two options are available - `backup`, and `restore`. This is a required parameter.
+*`--mode`*:: Specifies whether the command will perform a backup operation or a restore operation. The available options are:  `backup` (set by default), `restore`, and `verify`. You must define the `mode` option for `verify` or `restore` operations.
```

Comment:
Consider:
1. Use present simple in the first sentence: "Specifies whether the command performs ..."
2. Use a bulleted list with the four options.